modules/swagger-codegen/src/main/resources/cpprest/model-source.mustache

```diff
@@ -251,13 +285,14 @@ void {{classname}}::fromJson(web::json::value& val)
         {{/isDateTime}}
         {{^isDateTime}}
         if(!val[utility::conversions::to_string_t("{{baseName}}")].is_null())
-        {
-            {{{datatype}}} newItem({{{defaultValue}}});
-            newItem->fromJson(val[utility::conversions::to_string_t("{{baseName}}")]);
-            {{setter}}( newItem );
+        {        	
```

Comment:
spaces are used here not tabs

Reply:
also why is there unnecessary whitespace?

Reply:
I removed all tabs and (hopefully) all whitespace.